tv.plex.PlexHTPC.yml

```diff
@@ -0,0 +1,109 @@
+app-id: tv.plex.PlexHTPC
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/Plex.sh
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --device=all
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-music:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download:create
+  - --filesystem=xdg-cache:rw
+  - --filesystem=xdg-config:rw
+  - --filesystem=xdg-data:rw
+  - --filesystem=home:ro
+  - --talk-name=org.freedesktop.PowerManagement
+  - --talk-name=org.freedesktop.ScreenSaver
+modules:
+  - name: plex-icons
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID
+      - install -Dm664 $FLATPAK_ID.png /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+    sources:
+      - type: dir
+        path: .
+
+  - name: libjpeg # with libjpeg.so.8
```

Comment:
the libjpeg in the runtime isn't enough?

Reply:
Maybe a different ABI version? It definitely wouldn't pick it up from the runtime until I added it. Same goes for libwebp.

Reply:
webp 0.5.1 is SIX YEARS old.

Reply:
I'm not able to replace libwebp with a newer version. We are building Qt 5.15.3 from source and it vendors an old version. 0.5.1 seems to be the last version with ABI 6 (I could be wrong, but that's what I found).

Reply:
Because it doesn't install it for some reason and cqtdeployer doesn't collect it.

Reply:
I'm resolving this because I'm unable to do anything about it right now. It is noted that it would be beneficial to ensure that the binary package works standalone.